samples/matter/template/README.rst

```diff
@@ -177,7 +177,7 @@ When you have built the sample and programmed it to your development kit, it aut
 At this point, you can press **Button 1** for six seconds to initiate the factory reset of the device.
 
 .. note::
-    If you are new to Matter, commission the Matter device using the Mobile Controller for Android (CHIP Tool for Android) when :ref:`setting up the Matter development environment <ug_matter_configuring_mobile>`.
+    If you are new to Matter, commission the Matter device using the CHIP Tool for Linux/macOS when :ref:`setting up the Matter development environment <ug_matter_gs_testing_thread_separate_otbr_linux_macos>`.
```

Comment:
I don't think this note is useful now, maybe we should remove it. As far as I remember it was suggestion like "Hey, if you are new to Matter use Android CHIPTool, because it's easier compared to command line tool". Now we just recommend using CHIPTool here, but in section below we link once again to the same page, where cmd CHIPTool is in fact the only available tool :)

Reply:
Good point, that note can be removed.

Reply:
Agreed, please remove the note.